src/components/ColorPicker/components/ColorPickerContent/ColorPickerColorsGrid.jsx

```diff
@@ -21,12 +21,26 @@ export const ColorPickerColorsGrid = React.forwardRef(
       SelectedIndicatorIcon,
       colorSize,
       tooltipContentByColor,
-      colorShape
+      colorShape,
+      showColorNameTooltip: showColorNameTooltip
     },
     ref
   ) => {
     const getItemByIndex = useCallback(index => colorsToRender[index], [colorsToRender]);
 
+    const calculateColorTooltip = color => {
+      if (tooltipContentByColor && tooltipContentByColor[color]) return tooltipContentByColor[color];
+      else {
+        return showColorNameTooltip ? formatColorNameForTooltip(color) : {};
```

Comment:
is it intentional to return an empty object in case that `showColorNameTooltip` is falsy? I'd expect it to be `undefined`, no?